build/pkgs/patchelf/distros/conda.txt

```diff
@@ -0,0 +1 @@
+patchelf
```

Comment:
This one doesn't seem to exist, see https://github.com/sagemath/sage/actions/runs/6598197145/job/17925932752.

Reply:
It only exists in anaconda, not conda-forge. See #35593.